src/anemoi/models/layers/block.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         num_heads: int,
         activation: str,
         window_size: int,
+        grid_lat_coslon_sinlon: Tensor = None,
```

Comment:
annotation: `Tensor` -> `Optional[Tensor]`?
(below, too)

Reply:
Thank you for spotting this. Resolved by [df25841](https://github.com/ecmwf/anemoi-models/pull/96/commits/df258416c87f88a7cf7d90076c241ea26a6a3dc3).